jobs/admin.py

```diff
@@ -11,10 +13,22 @@ class JobAdmin(admin.ModelAdmin):
             'fields': ('title', 'company', 'location'),
         }),
         ('Detay Bilgiler', {
-            'fields': ('url', 'date_created', 'description'),
+            'fields': ('is_closed', 'url', 'date_created', 'description'),
         }),
     )
 
-    list_display = ('title', 'company')
-    list_filter = ('company',)
+    list_display = ('title', 'company', 'is_closed')
+    list_filter = ('is_closed', 'company',)
     search_fields = ('title', 'company')
+
+    def get_fieldsets(self, request, obj=None):
```

Comment:
Güzel düşünmüşsün. Bence şu an için sistemi olabildiğince basit tutmakta fayda var. Eğer istersen iş ilanı eklemeyi tamamen devre dışı bırakabiliriz admin panelinde. İmplementasyonu da daha kısa olur muhtemelen:

```py
def has_add_permission(self, request):
    return False
```